code/modules/power/heavy_emitter.dm

```diff
@@ -0,0 +1,261 @@
+/obj/machinery/power/heavy_emitter
+	name = "Heavy Emitter Base"
+	desc = "Message an admin if you see this!"
+	icon = 'icons/obj/heavy_emitter.dmi'
+	icon_state = "centre_off"
+	anchored = FALSE
+	density = TRUE
+	payment_department = ACCOUNT_ENG
+
+/obj/machinery/power/heavy_emitter/wrench_act(mob/living/user, obj/item/I)
+	. = ..()
+	if(anchored)
+		return FALSE
+	I.play_tool_sound(src, 50)
+	to_chat(user, "<span class='notice'>You start to rotate [src].</span>")
+	if(do_after(user,10 SECONDS,FALSE,src))
+		setDir(turn(dir,-90))
+	return TRUE
+
+/obj/machinery/power/heavy_emitter/examine(mob/user)
+	. = ..()
+	if(anchored)
+		. += "It is welded to the floor"
+
+/obj/machinery/power/heavy_emitter/welder_act(mob/living/user, obj/item/I)
+	. = ..()
+	I.play_tool_sound(src, 50)
+	to_chat(user, "<span class='notice'>You start welding [src].</span>")
+	if(do_after(user,10 SECONDS,FALSE,src))
+		anchored = !anchored
+	return TRUE
+
+/obj/machinery/power/heavy_emitter/proc/check_part_connectivity()
+	return TRUE
+
+/obj/machinery/power/heavy_emitter/proc/turn_on()
+	return
+
+/obj/machinery/power/heavy_emitter/proc/turn_off()
+	return
+
+/obj/machinery/power/heavy_emitter/centre
+	name = "Heavy Emitter Core"
+	desc = "A dangerously unstable, military grade capacitor that eats power like it's candy, before releasing an incredibly potent burst of energy that can annihilate anything."
+	idle_power_usage = 500
+	active_power_usage = 2000
```

Comment:
What units is this in? how long does it take to drain a net/apc/inducer? Have you thought about it?

Reply:
it drains uses wire

Reply:
What unit is this power use in? How long does it take to drain a standard smes? is this level of power use significant enough to matter currently?

Reply:
no clue what units this is in, it isnt documented anywhere afaik. i know they are high numbers because i looked at diffrent machinery and figured out what would work well with it

Reply:
Have you tested the rate? Does it make a noticeable change to how you would interact with the thing in game?

Reply:
yes i testes it, it drains an enormous amount of energy. you physically cannoy run this thing on n2 sm

Reply:
it will quickly drain the whole station worth of energy if the sm is weak

Reply:
alright, works for me. I'll do a quick test tomorrow and we should be good

Reply:
*tomorrow

Reply:
Jesus christ